src/onboarding/index.ts

```diff
@@ -1,20 +1,7 @@
-export interface OnboardingExperimentParams {
```

Comment:
the return type is the same, but I don't think it needs to be duplicated here. it's inferred from the defaultValues field on the experiment config